plugins/outputs/websocket/websocket.go

```diff
@@ -92,7 +92,14 @@ func (w *WebSocket) Connect() error {
 
 	headers := http.Header{}
 	for k, v := range w.Headers {
-		headers.Set(k, v)
+		secret, err := v.Get()
+		if err != nil {
+			return err
```

Comment:
Please ease debugging by giving a hint which header failed
```suggestion
			return fmt.Errorf("getting header secret %q failed: %w", k, err)
```